server/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/SnapshotBlocksIT.java

```diff
@@ -114,11 +113,8 @@ public void testCreateSnapshotWithIndexBlocks() {
         logger.info("-->  creating a snapshot is blocked when an index is blocked for reads");
         try {
             enableIndexBlock(INDEX_NAME, SETTING_BLOCKS_READ);
-            assertBlocked(client().admin().cluster().prepareCreateSnapshot(REPOSITORY_NAME, "snapshot-2")
```

Comment:
Can't we use block level `METADATA_READ` to still test the same thing instead of removing the coverage for a block being found?

Reply:
prior to this change, we only checked `METADATA_READ` on the global level, not the index level (don't know why, but there we only checked `READ`). I did not want to make the block check in `TransportCreateSnapshotAction` more restrictive as it used to be, as this could possibly break some users. I'm not even sure if these blocks are appropriate for the snapshot/restore functionality at all.

Reply:
Ah ok, thanks for explaining :)